### DISCUSSION TITLE
Open calendar to current day

### DISCUSSION THREAD

**Discussion by antcorro** [YetiForceCompany/YetiForceCRM#15599] [How to]
Hello,

when I enter in Calendar and select "Day",  it do not point to current day, but to an arbitrary day.
For what I see it points always to the day after the last day I have visited. Please see the screentogif attached.
Is this a bug? or is there any configuration to point the Day to current day when I open Calendar?

![Peek 30-12-2021 23-06](https://user-images.githubusercontent.com/10244840/147791308-9b18eb7f-b3c4-439d-a9fb-620629b90e18.gif)

Thanks

**Comment by antcorro** [comment]
Hello,

any help on this?

Thanks

**Reply by apcloic** [reply]
@antcorro Did you check user, system and server time zone configuration ?

**Reply by antcorro** [reply]
mmm sorry I don't understand your question, I tried this on https://gitstable.yetiforce.com, so the user, system and server timezone configuration are privided by mantainers. You can try yourself and verify the behaviour described above.

**Reply by apcloic** [reply]
@antcorro
I've just tried on gitstable version, no issue on my side.
Regards,
